components/src/atoms/description-list/DescriptionList.html

```diff
@@ -0,0 +1,16 @@
+<dl class="list list-description">
+  <!--@slot Custom list content (Named slots)-->
```

Comment:
Please add a space between comment signs here and in other instances
`<!-- @slot Custom list content (Named slots) -->`